reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+
+In 2020, 185 members stepped down or were removed for inactivity (18
+months with 0 activity across any GitHub organization). The Kubernetes project 
+has around 1300 active members at any given time.   
+
+Joining the project's GitHub organizations is celebrated, but
+there are no regular means of celebration or recognition for those that join a
+SIG, community group, or [achievement for climbing the ladder].
+
+[elections]: https://github.com/kubernetes/steering/blob/master/elections.md#eligibility-for-voting
+[membership]: https://git.k8s.io/community/community-membership.md#member
+[emeritus status]: https://k8s.dev/docs/guide/owners/#emeritus
+[achievement for climbing the ladder]: https://github.com/kubernetes/test-infra/issues/11994
+[charter process]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-charter-template.md
+
+
+#### Areas of research
+
+- Automation around suggesting when contributors may be ready to move to the
+  next step in the lifecycle
+- Ways to promote a sustainable flow through the various stages of the lifecycle
+- Clarifying and/or simplifying definitions around the stages of the lifecycle
+- Prompts to assist contributors in knowing what they need to do to 'level up'
+  to the next stage in the lifecycle (e.g. prompts when you join a mailing list)
+
+
+### Growing a diverse group of OWNERs
+
+Growing on the contributor ladder to ownership doesn't happen immediately after
+your first contribution. Building trust is a key part of our values and is a
+step in the "contributor journey", called the Kubernetes [contributor
+ladder][ldr]. We need more diverse, trusted folks from all backgrounds to grow
+in these areas. Balancing being welcoming and identifying contributors to
+encourage to stick around at scale is difficult, especially with tens of
+thousands of casual contributors (yes, a small city) and many cultures that come
+together. 30 new leaders have stepped up to Chair and other roles in 2020, but
+bandwidth for the reviewers and approvers in many subprojects remains a
+challenge as does the diversity of those folks.
+
+The Groups utilize the following to find and grow contributors:
+
+- Consistent labeling of issues with `good-first-issue` and `help-wanted`. This
+  was reported as the most successful way of landing new contributors.
+- Intentionally creating space at meetings for new contributors to get involved
+  and/or a dedicated triage meeting that provide an overview of current 
+  priorities
+- Programs such as [Meet Our Contributors], an Office hours like space where
+  people can ask questions related to contributing.
+- Contributor events such as Contributor Summits, or Maintainer focused sessions
+  at KubeCon.
+- One-on-One sessions for dedicated contributors that have demonstrated a vested
+  interest in contributing and climbing the contributor ladder.
+- Group study groups for reviewers, approvers, and Chairs
```

Comment:
"Study groups"? What's the action (verb) here? ("Teaching people to be reviewers/approvers chairs"? "Mentoring?")